### ISSUE TITLE
PATH only under $params['context']['resource']['file']

### ISSUE THREAD

**Issue by betterapp** [BlackbitDigitalCommerce/pimcore-data-director#38]
In 2.8.5 under 
$params['context']['resource']['file']

was full path to asset with file name.

In 3.0.6 there is only full path WITHOUT file name.

Why ?

**Comment by BlackbitNeueMedien** [comment]
`$params['context']['resource']['file']` contains the used import source. This can be a folder, a glob expression, a single file or a URL etc.
If you want to know from which file a certain raw data item came from, please add a raw data field `__source`

Here is a quote from the documentation:
> You can specify a folder path as import source instead of a single file. In this case files in this folder (and subfolders) are imported subsequently (sorted by modification time - oldest first). Already imported files are stored in an `archive` folder within the import folder.
>
> In addition to the source file fields you can use the field `__source` to obtain the source file name. This can be useful if you want to process the source file name in the object import (e.g. to call a dependent import with the same import file).

**Comment by BlackbitNeueMedien** [comment]
Yes, I think so. Previously the resolved source file was listed there. But this is not correct because when you import data from several files in a folder `$params['context']['resource']['file']` cannot always contain the same file.

**Comment by betterapp** [comment]
![obraz](https://user-images.githubusercontent.com/7449124/177533173-33a72fa2-d776-4ce2-85bb-31e943b41bd7.png)

How can I access to __source from Result Callback Function ??

**Comment by betterapp** [comment]
I checked it allready - it is NULL

We use wildcard here:
![obraz](https://user-images.githubusercontent.com/7449124/177538225-9bf57330-25e6-4a9e-b9c4-73f2ced27194.png)

**Comment by BlackbitNeueMedien** [comment]
Oh, `__source` did not work for Excel-based dataports. Is fixed in 3.0.10.

**Comment by betterapp** [comment]
is it released with tag or still in development ?

**Comment by BlackbitNeueMedien** [comment]
Is already released.
